.travis.yml

```diff
@@ -170,7 +170,12 @@ jobs:
     - name: "Elasticsearch TAS tests"
       if: commit_message !~ /\[skip search\]/
       script:
-        - travis_wait 30 mvn -B install -f tests/tas-elasticsearch/pom.xml -Pall-tas-tests -Denvironment=default -DrunBugs=false
+        - travis_wait 30 mvn -B install -f tests/tas-elasticsearch/pom.xml -Pall-tas-tests,elastic -Denvironment=default -DrunBugs=false
```

Comment:
Perhaps instead of profiles you can set the value of suiteXmlFile variable (-DsuiteXmlFile=...)
or have the profiles have different values instead of duplicating the maven-surefire-plugin config.

Reply:
That will make things easier to read, thanks. I'll do that.